internal/debug.go

```diff
@@ -0,0 +1,50 @@
+/*
+	Debug contains information about debug levels and define
+	debug statements
+*/
+package internal
+
+import (
+	"log"
+	"os"
+)
+
+var (
+	// logFlags    = log.Ldate | log.Ltime | log.Lshortfile
+	logFlags    = log.Ldate | log.Ltime
+	info = log.New(os.Stdout, "INFO: ", logFlags)
+	errorLog    = log.New(os.Stderr, "ERROR: ", logFlags)
+	warning     = log.New(os.Stderr, "WARNING: ", logFlags)
+)
+
+
+type DebugLogger interface {
+	LogInfo(string)
+	LogError(error)
+	LogWarning(string)
+	PrintVersion(string)
+}
+
+type Debug struct {
+	DebugLevel  bool
+}
+
+func (d Debug) LogInfo(message string) {
+	if d.DebugLevel {
+		info.Println(message)
+	}
+}
+
+func (d Debug) LogError(message error) {
+	errorLog.Println(message)
```

Comment:
Does `Println` take into account the type of output stream to use?

Reply:
`Println` simply prints the error message to stdout, as it does with `fmt`. Is there an additional need?

Reply:
@10eMyrT we may want this to print out to standard error, not standard out.

Reply:
I see. That's the best way. I will integrate it in the changes before next meeting.